openedx/features/course_experience/static/course_experience/js/CourseSock.js

```diff
@@ -57,8 +57,8 @@ export class CourseSock {  // eslint-disable-line import/prefer-default-export
 
       // Log open and close events
       const isOpening = $toggleActionButton.hasClass('active');
-      const logMessage = isOpening ? 'User opened the verification sock.'
-          : 'User closed the verification sock.';
+      const logMessage = isOpening ? 'edx.bi.course.sock.toggle_open'
+          : 'edx.bi.course.sock.toggle_close';
       Logger.log(
```

Comment:
According to [my notes on analytics](https://openedx.atlassian.net/wiki/spaces/~rraposa/pages/162251295/Analytics+and+Diagnostics), these client-side events using `Logger.log` go to the tracking log only, but not to GA.  I think we need to use `analytics.track`.  I'm not sure if any other changes are needed.  I imagine we do this in addition to the `Logger.log` calls, but not sure.  I also don't know how to document where client side analytics are being written to, but we should add that to the Confluence documentation to be clear.
@mulby Is this accurate? ^
FYI: @marcotuts